### PR TITLE
Added scrolling to subscription view.

### PR DESCRIPTION
**Pull request by omaruszczak-r7**
# Description
- Currently it's not possible to see the latest result of subscription if there are too many. Scrolling allows to see all subscription results on the page.
- Stops tabs at the bottom of the page from disappearing when subscription results go off-screen. 

# Testing
- Ran locally

### CODE REVIEW

File: packages/graphql-playground-react/src/components/Playground/SchemaFetcher.ts [hunk 1]
```diff
@@ -6,16 +6,16 @@ import { makeOperation } from './util/makeOperation'
 import { parseHeaders } from './util/parseHeaders'
 import { LinkCreatorProps } from '../../state/sessions/fetchingSagas'
 import * as LRU from 'lru-cache'
-
-export interface CsrfHeader {
```
Comment (rwilliams-r7) [inline packages/graphql-playground-react/src/components/Playground/SchemaFetcher.ts]: ??

File: packages/graphql-playground-react/src/components/Playground/EditorWrapper.tsx [hunk 2]
```diff
@@ -913,24 +908,12 @@ const GraphqlContainer = styled.div`
   width: 100%;
 `
 
-export class Container extends React.PureComponent {
```
Comment (erran-r7) [inline packages/graphql-playground-react/src/components/Playground/EditorWrapper.tsx]: Revert all non-CSS changes.

File: packages/graphql-playground-react/src/components/Playground/SchemaFetcher.ts [hunk 3]
```diff
@@ -169,7 +159,9 @@ export class SchemaFetcher {
         },
         error: err => {
           reject(err)
+          log(err)
```
Comment (erran-r7) [inline packages/graphql-playground-react/src/components/Playground/SchemaFetcher.ts]: Strip out all non-css changes please. Looks like a conflict merge/rebasing after the upgrade to Apollo's branch of playground.

**Review by erran-r7 — CHANGES_REQUESTED** [review]

**Review by erran-r7 — APPROVED** [review]

**Comment by erran-r7** [comment]
Let's do a squash & merge for this PR once we finish testing in our internal build. 😄

**Review by erran-r7 — APPROVED** [review]

**Pull request merged**